src/utilities/gql.ts

```diff
@@ -2521,6 +2604,38 @@ const gql = {
     }
   `,
 
+  SUB_MOST_RECENT_EXPANSION_FOR_SIMULATION_SEQS: `#graphql
+    subscription SubMostRecentExpansion {
+      ${Queries.EXPANDED_SEQUENCES} {
+        seq_id
+        expanded_sequence
+        simulation_dataset_id
+      }
+    }
+  `,
+
+  SUB_MOST_RECENT_EXPANSION_FOR_SIMULATION_SIMS: `#graphql
+    subscription SubMostRecentExpansion($planId: Int!) {
+      plan_by_pk(id: $planId) {
```

Comment:
There is a `Queries.PLAN` enum that can be used here instead of hardcoding `plan_by_pk`